src/main/java/parser/Parser.java

```diff
@@ -0,0 +1,244 @@
+package parser;
+
+import command.*;
```

Comment:
Referring to the [Coding Standard](https://se-education.org/guides/conventions/java/intermediate.html) under 

>   Imported classes should always be listed explicitly.

Wildcard import is listed under "Bad" example.